src/scripts/remove_transgene.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+"""
+Remove transgene entries from a regtools junctions.bed file, given
+a reference .bed file containing gene locus.
+"""
+
+import sys
+import argparse
+import os.path
+import subprocess as sp
+
+parser = argparse.ArgumentParser(
+    description="Remove transgene entries from a regtools junctions.bed file, given \
+    a reference .bed file containing gene locus.",
+    epilog="Usage eg: python remove_transgene.py <hg38.refGene_gene_longest.bed> <junctions.bed> <output.bed>"
+)
+
+parser.add_argument("refgene_path", help="Path to reference .bed file containing gene loci.")
+parser.add_argument("junctions_path", help="Path to .bed file containing junctions from regtools.")
+parser.add_argument("outname", help="Full path to output file.")
+
+args = parser.parse_args()
+
+if not os.path.exists(args.refgene_path):
+    print "Reference .bed file does not exist! Exiting..."
+    exit()
```

Comment:
Can you change this to `exit(1)`? I forgot to add exit codes when I wrote this script.